CI-7271 documenting our upcoming mTLS support (#178)
AdobeDocs/adobe-io-events: src/pages/guides/index.md

## Security Considerations

<InlineAlert variant="info" slots="text"/>

> Adobe strongly encourages securing your webhook deliveries to avoid processing malicious request.

Reviewer: nitpik: it should be request**s**

Indeed, as your webhook URL must necessarily be accessible from the open internet, 
third-party actors can send forged requests to it, tricking your application into handling fake events.

> To prevent this from happening, Adobe I/O Events offers two options for securing your webhook deliveries:

Reviewer: instead of "to prevent this from happening" -> "To mitigate this risk"

* Adobe I/O Events Digital Signature 
* Adobe I/O Events  `mTLS` support

> Both options are described below, pick just one, implementing both would be a waste of resources.

Reviewer: im wondering if we should say wast of resources. does not sound very professional. 
may be rephrase it to : 

You should choose one of these options to implement. Using both is unnecessary and would only consume additional resources without added benefit.

### Adobe I/O Events Digital Signature

> - Adobe I/O Events sends an additional field of `recipient_client_id` as part of your event payload.

Author: We will do this change properly in a separate PR

<InlineAlert variant="info" slots="text"/>
Kindly note that this digital signature verification process comes out-of-the-box for I/O Runtime actions, and no action is required on that end.

### Adobe I/O Events mTLS support

Instead of leveraging the Adobe I/O Events Digital Signature, you may choose to secure your webhook using `mTLS`.

> `mTLS` (mutual Transport Layer Security) will ensure that only Adobe I/O Events can send events to your webhook.

Author: well, this was done on purpose the m of mutual is lower case because the m of mTLS is lower case too

  * it can be validated using the CA’s public key
  * if revoked, the certificate will be added to the Certificate Revocation List (CRL) of the CA
* Adobe I/O Events certificate Common Name (CN) and Subject Alternative Names (SAN) will remain the same across renewal, and can be used as an additional layer of ownership validation if you wish to do so.

> * in order to debug this mTLS handshake, you can use the `Debug Tracing` UI, see our [Debug Tracing documentation](../support/tracing.md) for more details.

Reviewer 2: ```suggestion
* In order to debug this mTLS handshake, you can use the [Debug Tracing](../support/tracing.md) tab in Developer Console.
```
